File: Front/src/classes/model/QrCode/Cliente.js
export default class Cliente{
    constructor(ClienteId, NomeCliente, Ativo, ApiPertencente = 2){
        this.ClienteId = ClienteId;
        this.NomeCliente = NomeCliente;
        this.Ativo = Ativo;
        this.ApiPertencente = ApiPertencente;
    }
} 

File: MarcaQr/Dominio/Entidades/Cliente.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Dominio.Entidades
{
    public class Cliente
    {
        public int ClienteId { get; set; }
        public string NomeCliente { get; set; }
        public bool Ativo { get; set; }
        public int ApiPertencente { get; set; }
        public DateTime DataCadastro { get; set; }
        public virtual IEnumerable<Usuario> Usuarios { get; set; }
    }
}
